cranetoolbox/importTools/transform.py

```diff
@@ -111,18 +124,15 @@ def filter_lighten_chunk(chunk, opts: TransformationOptions) -> (List[dict], int
             # This is dirty and redudant but I don't know of a better method.
```

Comment:
Could you delete this comment since you've improved the conditions check? :)

Reply:
okay